src/ff/prime_field.rs

```diff
@@ -167,15 +180,30 @@ pub struct Fp31(<Self as Field>::Integer);
 impl Field for Fp31 {
     type Integer = u8;
     const PRIME: Self::Integer = 31;
+    const PRIME_TO_INTMAX_DELTA: Self::Integer = 225;
     const ZERO: Self = Fp31(0);
     const ONE: Self = Fp31(1);
 }
 
 field_impl! { Fp31 }
 
+#[derive(Clone, Copy, PartialEq)]
+#[cfg_attr(feature = "enable-serde", derive(Serialize, Deserialize))]
```

Comment:
I believe we don't need this anymore. There are two special methods to serialize and deserialize a field. I forgot to remove it from `Fp31` :(